### PR TITLE
Enhancing Vespa’s Embedding Management Capabilities

### PR DESCRIPTION
**Pull request by kkraune**
- I had to get creative on the table col headers, @jobergum, to fit nicely on the page, please review

FYI @bratseth @frodelu @johans1

### CODE REVIEW

**Review by jobergum — APPROVED** [review]

File: _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+GPU support in Vespa also unlocks using larger and more powerful embedding models
+while maintaining low serving latency and cost-effectiveness.
+
+GPU acceleration is automatically enabled in Vespa Cloud for instances where GPU(s) is available. 
+Configure your stateless Vespa container cluster with a [GPU resource in services.xml](https://cloud.vespa.ai/en/reference/services#gpu).
+For open-source Vespa, specify the GPU device using the
+[embedder ONNX configuration](https://docs.vespa.ai/en/reference/embedding-reference.html#embedder-onnx-reference-config).  
+
+## Vespa Model Hub Updates
+To further enrich the Vespa Cloud ecosystem,
```
Comment (bratseth) [inline _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md:87]:
```suggestion
To make it easier to create embedding applications,
```

File: _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+we introduce new state-of-the-art text embedding models on the [Vespa Model Hub](https://cloud.vespa.ai/en/model-hub) for
```
Comment (bratseth) [inline _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md:88]:
```suggestion
we have added new state-of-the-art text embedding models on the [Vespa Model Hub](https://cloud.vespa.ai/en/model-hub) for 
```

File: _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+developers do not need to worry about scaling with changes in inference traffic volume.
+
+Vespa Cloud also allows bringing your own models using the [HuggingFace Embedder](https://docs.vespa.ai/en/embedding.html#huggingface-embedder) 
+with model files submitted in the application package. In Vespa Cloud, inference with embedding models is 
+automatically accelerated with GPU if the application uses [Vespa Cloud GPU instances](https://blog.vespa.ai/gpu-accelerated-ml-inference-in-vespa-cloud/).
+Read more on the [Vespa Cloud model hub](https://cloud.vespa.ai/en/model-hub).
+
+
+## Summary
+The improved Vespa embedding management options offer a significant leap forward in Vespa’s embedding capabilities,
```
Comment (bratseth) [inline _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md:148]:
```suggestion
The improved Vespa embedding management options offer a significant leap in capabilities for anybody working with embeddings in online applications,
```

File: _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+---
+
+![Decorative
+image](/assets/2023-06-13-enhancing-vespas-embedding-management-capabilities/vnwayne-fan-Zqmia99hgF8-unsplash.jpg)
+<p class="image-credit">Photo by 
+<a href="https://unsplash.com/@vnwayne?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">vnwayne fan</a> 
+on <a href="https://unsplash.com/photos/Zqmia99hgF8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></p>
+
+We are thrilled to announce significant updates to Vespa’s support for inference with text embedding models
+that maps texts into vector representations.
```
Comment (bratseth) [inline _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md:21]:
```suggestion
that maps texts into vector representations: General support for Huggingface models including multi-lingual embedding, embedding inference on GPUs, and new recommended models available on the Vespa Cloud model hub.
```

**Review by bratseth — APPROVED** [review]
Great!

File: _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+## Embedder Models from Huggingface
+Vespa now comes with generic support for embedding models hosted on [Huggingface](https://huggingface.co/).
+
+With the new Huggingface Embedder functionality,
+developers can export embedding models from Huggingface
+and import them in [ONNX](https://onnx.ai/) format in Vespa for accelerated inference close to where the data is created:
+
+```
+<container id="default" version="1.0">
+    <component id="e5" type="hugging-face-embedder">
```
Comment (bjorncs) [inline _posts/2023-06-13-enhancing-vespas-embedding-management-capabilities.md:52]:
```suggestion
    <component id="my-embedder-id" type="hugging-face-embedder">
```

**Review by bjorncs — APPROVED** [review]

**Pull request merged**